README-fr.md

```diff
@@ -0,0 +1,83 @@
+
+Une fois votre pull request créée, un examinateur de Kubernetes se chargera de vous fournir une revue claire et exploitable.
+En tant que propriétaire de la pull request, **il est de votre responsabilité de modifier votre pull request pour tenir compte des commentaires qui vous ont été fournis par l'examinateur de Kubernetes.** 
+Notez également que vous pourriez vous retrouver avec plus d'un examinateur de Kubernetes pour vous fournir des commentaires ou vous pourriez finir par recevoir des commentaires d'un autre examinateur que celui qui vous a été initialement affecté pour vous fournir ces commentaires.
+De plus, dans certains cas, l'un de vos examinateur peut demander un examen technique à un[examinateur technique de Kubernetes](https://github.com/kubernetes/website/wiki/Tech-reviewers) au besoin.
+Les examinateurs feront de leur mieux pour fournir une revue rapidement, mais le temps de réponse peut varier selon les circonstances.
+
+Pour plus d'informations sur la contribution à la documentation Kubernetes, voir :
+
+* [Commencez à contribuer](https://kubernetes.io/docs/contribute/start/)
+* [Apperçu des modifications apportées à votre documentation](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Utilisation des modèles de page](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Traduction de la documentation Kubernetes](https://kubernetes.io/docs/contribute/localization/)
+
+## Exécuter le site localement en utilisant Docker
+
+La façon recommandée d'exécuter le site web Kubernetes localement est d'utiliser une image spécialisée[Docker](https://docker.com) qui inclut le générateur de site statique[Hugo](https://gohugo.io).
+
+> Si vous êtes sous Windows, vous aurez besoin de quelques outils supplémentaires que vous pouvez installer avec [Chocolatey](https://chocolatey.org). `choco install install make`
+
+> Si vous préférez exécuter le site Web localement sans Docker, voir[Exécuter le site localement avec Hugo](#running-the-site-locally-using-hugo) ci-dessous.
+
+Si vous avez Docker [up and running](https://www.docker.com/get-started), construisez l'image Docker `kubernetes-hugo' localement:
+
+```bash
+make docker-image
+```
+
+Une fois l'image construite, vous pouvez exécuter le site localement :
+
+```bash
+make docker-serve
+```
+
+Ouvrez votre navigateur à l'adresse: http://localhost:1313 pour voir le site.
+Lorsque vous apportez des modifications aux fichiers sources, Hugo met à jour le site et force le navigateur à rafraîchir la page.
+
+## Exécuter le site localement en utilisant Hugo
+
+Voir la [documentation officielle Hugo](https://gohugo.io/getting-started/installing/) pour les instructions d'installation Hugo.
+Assurez-vous d'installer la version Hugo spécifiée par la variable d'environnement `HUGO_VERSION' dans le fichier[`netlify.toml`](netlify.toml#L9).
```

Comment:
Change `\`HUGO_VERSION'` to `\`HUGO_VERSION\``

Reply:
done